GithubLearning.html

```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>GitHubLearnings</title>
+</head>
+<body>
+    <p>
+        This command is used to start a new repository : git init
+        <br>
+        This command is used to obtain a repository from an existing URL : git clone [url] 
+        <br>
+        This command lists all the files that have to be committed.: git status
+        <br>
+        This command adds a file to the staging area.: git add [file]
+         <br>
+         This command records or snapshots the file permanently in the version history.: git commit -m “[ Type in the commit message]”
+         <br>
+         This command sends the branch commits to your remote repository. :git push [variable name] [branch]
+         <br>
+        This command fetches and merges changes on the remote server to your working directory.: git pull [Repository Link]
+        <br>  
+        This command lists all the local branches in the current repository.: git branch
+        <br>
+        This command creates a new branch : git branch [branch name]
```

Comment:
To create new branch we use -b 
cmd: **git checkout -b [branchName]**

To switch branch we use
cmd: **git checkout [branchName]**